tests/runall.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+[ ! -f /opt/kselftests/run_kselftest.sh ] && echo "Please install kselftests to /opt/kselftests first" && exit 1
+[ ! -f /opt/ltp/runltp ] && echo "Please install ltp to /opt/ltp first" && exit 1
+
```

Comment:
Could you please add script or document on how to build and install ltp and kselftests? We'd like to make the test as automatic as possible.

Reply:
Done